overview/content-types/code-snippet.nix

```diff
@@ -36,7 +36,7 @@ in
             {{ include_code("${self.language}", "${self.filepath}" ${optionalString self.relative ", relative_path=True"}) }}
             <div class="code-buttons">
               ${optionalString self.downloadable ''
-                <a class="button download" href="${self.filepath}" download>Download</a>
+                <a class="button download" href="default.nix" download>Download</a>
```

Comment:
Couldn't this simply take `self.filepath.name`?

Reply:
Yup, you're right